cmd/run/run.go

```diff
@@ -222,15 +222,17 @@ func NewRunCommand() *cobra.Command {
 
 	flags.Uint32("listUsers-max-results", defaultConfig.ListUsersMaxResults, "the maximum results to return in ListUsers API responses. If 0, all results can be returned")
 
+	flags.Uint32("check-cache-limit", defaultConfig.CheckCache.Limit, "if check-query-cache-enabled or check-iterator-cache-enabled, this is the size limit of the cache")
+
 	flags.Bool("check-iterator-cache-enabled", defaultConfig.CheckIteratorCache.Enabled, "enable caching of datastore iterators of Check requests.")
```

Comment:
Are we explaining somewhere in the docs how "caching of datastore iterators" works, what's the benefit, and when to enable them? 

Why don't we have as single flag for all caches? How does the user decide which one to enable?

Reply:
>Are we explaining somewhere in the docs how "caching of datastore iterators" works, what's the benefit, and when to enable them?

Yes, please review https://github.com/openfga/openfga.dev/pull/886.

>Why don't we have as single flag for all caches? How does the user decide which one to enable?

I don't have a good answer. While writing docs I realized that this is too cumbersome to explain, which probably means it's also cumbersome to use. We _should_ have just one caching flag.